code/modules/mob/living/life.dm

```diff
@@ -98,6 +98,10 @@
 	var/loc_temp = get_temperature(environment)
 	var/temp_delta = loc_temp - bodytemperature
 
+	if(istype(loc, /obj/structure/closet))
+		var/obj/structure/closet/current_locker = loc
+		temp_delta *= (1 - current_locker.thermal_insulation)
```

Comment:
Again these sort of things aren't good to have. Closet code should be in closets not in mobs.